### PR TITLE
docs: remove https://angular.io from internal links

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
PR #36601 itroduces icons on all links if the link contains https:// or http:// but there were some internal links left which contained https://angular.io. Removed https://angular.io from all these links.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Some internal links contain https:// which added external links icon

Issue Number: N/A

## What is the new behavior?
Removed https://

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/updating.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ To check your app's version of Angular: From within your project directory, use
 {@a checking-version-angular}
 ## Finding the current version of Angular
 
-The most recent stable released version of Angular appears in the [Angular documentation](https://angular.io/docs "Angular documentation") at the bottom of the left side navigation. For example, `stable (v5.2.9)`.
```
Comment (petebacondarwin) [inline aio/content/guide/updating.md:40]: I wonder if we want to keep this one?
For example when looking at archived docs (say `https://v8.angular.io`) we would still want the link to go to the latest docs, right?

**Review by petebacondarwin — APPROVED** [review]
Nice catch @ajitsinghkaler - I have approved but marked with cleanup since I "think" we should revert that one change...

File: aio/content/guide/updating.md (hunk 1 above)
Comment (ajitsinghkaler) [reply aio/content/guide/updating.md:40]: Yes, you're right

**Comment by ajitsinghkaler** [comment]
@petebacondarwin made the changes

**Review by petebacondarwin — APPROVED** [review]

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**